javascript/javascript1/week1/task2.js

```diff
@@ -0,0 +1,17 @@
+console.log("Goodboy-Oldboy!") // Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2017;
+
+let dogYearOfFuture = 2027;
+
+let inHumanYear = dogYearOfFuture - dogYearOfBirth;
+
+let dogYear = (dogYearOfFuture - dogYearOfBirth) * 7;
+
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+    console.log ("Your dog will be " + dogYear + " dog years old in " + dogYearOfFuture)
```

Comment:
you can use `${}`( enclosed between backtick. go through this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals ) to separate the variable. like this:console.log (`Your dog will be  ${dogYear}  dog years old in  ${dogYearOfFuture}`)
an easy way other way to combine string and variable(just a suggestion not mandatory!!).